File: 240408/사칙연산 함수/quadratic-operations-function.py
def add(a, b):
    return a + b

def diff(a, b):
    return a - b

def multiply(a, b):
    return a * b

def divide(a, b):
    return a // b

a, o, c = input().split()
a, c = int(a), int(c)
if o in ['+', '-', '/', '*']:
    if o == '+':
        print(a, o, c, "= {}".format(add(a, c)))
    elif o == '-':
        print(a, o, c, "= {}".format(diff(a, c)))
    elif o == '*':
        print(a, o, c, "= {}".format(multiply(a, c)))
    else:
        print(a, o, c, "= {}".format(divide(a, c)))
else:
    print(False)

File: 251216/사칙연산 함수/quadratic-operations-function.cpp
#include <iostream>
#include <string>
using namespace std;

int a;
int c;
string o;

int add(int a, int b);
int diff(int a, int b);
int multiply(int a, int b);
int divide(int a, int b);

int main() 
{
    cin >> a >> o >> c;

    // Please write your code here.
    if (o == "+" || o == "-" || o == "*" || o == "/") 
    {
        if (o == "+") 
        {
            cout << a << " " << o << " " << c << " = " << add(a, c);
        } 
        else if (o == "-") 
        {
            cout << a << " " << o << " " << c << " = " << diff(a, c);
        } 
        else if (o == "*") 
        {
            cout << a << " " << o << " " << c << " = " << multiply(a, c);
        } 
        else 
        { // "/"
            cout << a << " " << o << " " << c << " = " << divide(a, c);
        }
    } 
    else 
    {
        cout << "false";
    }

    return 0;
}

int add(int a, int b)
{
    return a + b;
}

int diff(int a, int b)
{
    return a - b;
}

int multiply(int a, int b)
{
    return a * b;
}

int divide(int a, int b)
{
    return a / b;
}
